spanner/google/cloud/spanner_v1/streamed.py

```diff
@@ -137,7 +137,10 @@ def __iter__(self):
         iter_rows, self._rows[:] = self._rows[:], ()
         while True:
             if not iter_rows:
-                self._consume_next()  # raises StopIteration
+                try:
+                    self._consume_next()  # raises StopIteration
```

Comment:
Probably don't need the comment, given the `try:...except StopIteration:`  :).

Reply:
Fair point.